quint/src/runtime/impl/runtimeValue.ts

```diff
@@ -712,8 +714,14 @@ abstract class RuntimeValueBase implements RuntimeValue {
     if (this instanceof RuntimeValueStr && other instanceof RuntimeValueStr) {
       return this.value === other.value
     }
-    if (this instanceof RuntimeValueTupleOrList && other instanceof RuntimeValueTupleOrList) {
-      return this.kind === other.kind && this.list.equals(other.list)
+    if (this instanceof RuntimeValueList && other instanceof RuntimeValueTuple) {
+      return false
+    }
```

Comment:
You can safely remove this since this scenario (one list and one tuple) won't match any of the other patterns, and the default return is false (when no pattern is matched - line 764)
```suggestion
```